Prepare for stack trace change in android (#117)
microsoft/appcenter-sdk-cordova: CHANGELOG.md

## Version 0.4.2

### App Center Crashes

#### Android

> - **[Bug Fix]** Fixes a security issue in the way native Android crashes are processed. The exception message will now be `null` and the exception stack trace might look slightly different.

Reviewer: ```suggestion
- **[Behavior change]** Fix a security issue in the way native Android crashes are processed. As a result, the exception stack trace for crashes that occurred in Java code is the raw Java stack trace and the associated exception message is now `null`.
```

> 

Reviewer 2: ```suggestion
_

```

> ## Version 0.4.2

Reviewer: ```suggestion
## Version 0.5.0 (Under development)
```

### App Center Crashes

#### Android

> - **[Behavior change]** Fix a security issue in the way native Android crashes are processed. As a result, the exception stack trace for crashes that occurred in Java code is the raw Java stack trace and the associated exception message is now `null`.

Reviewer 3: ```suggestion
- **[Behavior change]** Fix a security issue in the way native Android crashes are processed. As a result, the exception stack trace for crashes that occurred in Java code is now the raw Java stack trace and the associated exception message is now `null`.
```
